devicetypes/heltun/heltun-rs01-switch.src/heltun-rs01-switch.groovy

```diff
@@ -0,0 +1,560 @@
+			1: "Restore Last State"
+	], paramNum: 70, size: 1, default: "0", type: "enum"],    
+
+[title: "Relay Inverse Mode", description: "The values in this Parameter specify the relays that will operate in inverse mode. Relays can operate in an inverse mode in two different ways: 1. When the first and the second relays are connected to two different external switches. In this case, after pressing a button, the corresponding relay connected to that button will toggle its state (ON to OFF or OFF to ON), and the other relay will be switched OFF. 2.	When two relays are connected to the same external switch. In this case, the relays will operate in roller shutter mode and their behavior will follow these four cycles: a - 1st press of button: the first relay will be switched ON, the second relay will be switched OFF, b - 2nd press of button: both relays will be switched OFF, c - 3rd press of button: the second relay will be switched ON, the first relay will be switched OFF, d - 4th press of button: both relays will be switched OFF. ≡ Note: In this mode, both relays cannot be switched ON at the same time (i.e. simultaneously). ≡ Note: Switching OFF one relay will always operate before switching ON another relay to prevent both relays from being ON at the same time.",
+ name: "Group 1", options: [
+			0: "Disabled",
+			12: "1st & 2nd Relay",
+			13: "1st & 3rd Relay",
+			14: "1st & 4th Relay",
+			15: "1st & 5th Relay",
+			23: "2nd & 3rd Relay",
+			24: "2nd & 4th Relay",
+			25: "2nd & 5th Relay",
+			34: "3rd & 4th Relay",
+			35: "3rd & 5th Relay",
+			45: "4th & 5th Relay"
+	], paramNum: 101, size: 1, default: "0", type: "enum"],
+    
+[name: "Group 2", options: [
+			0: "Disabled",
+			12: "1st & 2nd Relay",
+			13: "1st & 3rd Relay",
+			14: "1st & 4th Relay",
+			15: "1st & 5th Relay",
+			23: "2nd & 3rd Relay",
+			24: "2nd & 4th Relay",
+			25: "2nd & 5th Relay",
+			34: "3rd & 4th Relay",
+			35: "3rd & 5th Relay",
+			45: "4th & 5th Relay"
+	], paramNum: 102, size: 1, default: "0", type: "enum"], 
+    
+[title: "Energy Consumption Meter Consecutive Report Interval", description: "When the device is connected to the gateway, it periodically sends reports from its energy consumption sensor even if there is no change in the value. This parameter defines the interval between consecutive reports of real time and cumulative energy consumption data to the gateway",
+ name: "Selected Energy Report Interval in minutes", paramNum: 141, size: 1, default: 10, type: "number", min: 1 , max: 120, unit: "min"],
+
+[title: "Control Energy Meter Report", description: "This Parameter determines if the change in the energy meter will result in a report being sent to the gateway. Note: When the device is turning ON, the consumption data will be sent to the gateway once, even if the report is disabled.",
+ name: "Sending Energy Meter Reports", options: [
+ 			0: "Disabled",
+			1: "Enabled"
+	], paramNum: 142, size: 1, default: "1", type: "enum"]
+ 
+]}
```

Comment:
nit: indentations in entire block (+ 1 tab)
nit: there are some unnecessary empty spaces at the end of some lines